sources/Renderer/Metal/MTModuleInterface.mm

```diff
@@ -9,30 +9,55 @@
 #include "MTRenderSystem.h"
 
 
-extern "C"
+namespace LLGL
 {
+	namespace ModuleMetal
```

Comment:
I never had issues with mixed tabs and spaces, since i always use external tools. I ran a bash script to fix that , no idea why the tab was not fixed.